backend/value-resolver/src/main/kotlin/com/ritense/valueresolver/ValueResolverServiceImpl.kt

```diff
@@ -303,5 +311,6 @@ class ValueResolverServiceImpl(
 
     companion object {
         const val DELIMITER = ":"
+        private val prefixRegex = Regex("^[A-Za-z_-]+$") // no numbers allowed
```

Comment:
What issues does this solve?
You can't just invent new requirements for the prefix. This could be breaking for custom value resolvers.